platform-sdk/swirlds-config-extensions/src/main/java/com/swirlds/config/extensions/sources/PropertyFileConfigSource.java

```diff
@@ -55,6 +57,9 @@ public PropertyFileConfigSource(@NonNull final Path filePath, final int ordinal)
      */
     @NonNull
     protected BufferedReader getReader() throws IOException {
+        if (!FileSystem.waitForPathPresence(filePath)) {
+            throw new FileNotFoundException("File not found: " + filePath);
+        }
```

Comment:
I feel like this wait should happen before this instance is constructed

Reply:
I still think none of these config classes should be waiting, we should do the wait before we try to initialize the config

Reply:
If I move the logic, it results in duplicated code in multiple locations. Should we disregard being `DRY` simply to avoid the configuration logic from waiting for file existence?

Reply:
I've gone ahead and moved the logic to the 4 known immediate callers of `PropertyFileConfigSource` not including test code locations.